vimeo-networking/src/main/java/com/vimeo/networking/AccountStore.java

```diff
@@ -24,25 +24,37 @@
 
 import com.vimeo.networking.model.VimeoAccount;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
 /**
  * Interface responsible for handling the creation, deletion, and loading of Vimeo accounts on the client.
  * <p/>
  * Created by hanssena on 4/27/15.
  */
 public interface AccountStore {
 
+    /**
+     * Load the previously saved {@link VimeoAccount} and return it
+     *
+     * @return null if no VimeoAccount could be loaded
+     */
+    @Nullable
     VimeoAccount loadAccount();
 
     /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
+     * Saves a {@link VimeoAccount} for loading later.
+     *
+     * @param vimeoAccount the VimeoAccount to save, should not be null when calling this method
+     * @param accountName  the <i>name</i> of the account that helps uniquely identify it in account
+     *                     stores, such as the Android AccountManager
      */
-    @Deprecated
-    void saveAccount(VimeoAccount vimeoAccount, String email, String password);
+    void saveAccount(@NotNull VimeoAccount vimeoAccount, String accountName);
```

Comment:
Again, the reason that I didn't add it is this: if the VimeoAccount represents a client credentials grant, then the accountName should be null, and we would want to show this using the Nullable annotation. 
If the VimeoAccount represents an authenticated user, than the accountName should not be null - but we shouldn't use the NotNull annotation here because of the fact that it would be null in the aforementioned case. 

I think adding either annotation would be misleading - many times a user looks at the signature of the method to determine what is required to be passed in. I think leaving it off in a scenario here is helpful - as a consumer of the method I would try and look at the javadoc or the method more closely to determine if it is needed and in what scenarios then.

I'd like to leave annotations off of method parameters like this until we discuss in Android sync the preferred approach. And since all of these files will be refactored before a 2.0.0 release, we have more than plenty of time to drop the annotations in (or not, depending on the outcome of our discussion).